content/en/docs/advanced-tasks/catalog-update-formulary.md

```diff
@@ -0,0 +1,382 @@
+
+
+## Examples
+For brevity, all formulae will refer to the same example, for semver template and FBC.  For convenience, all semver bundle image pullspecs will express versions which match the bundle version (instead of SHAs). 
+
+### semver template example
+Formulae presume the following content is saved to the file `semver.yaml`
+```yaml
+schema: olm.semver
+generatemajorchannels: false
+generateminorchannels: true
+candidate:
+  bundles:
+  - image: quay.io/organization/testoperator:v1.0.0
+  - image: quay.io/organization/testoperator:v1.0.1
+  - image: quay.io/organization/testoperator:v1.1.0
+fast:
+  bundles:
+  - image: quay.io/organization/testoperator:v1.0.1
+  - image: quay.io/organization/testoperator:v1.1.0
+stable:
+  bundles:
+  - image: quay.io/organization/testoperator:v1.0.1
+  ```
+
+
+### FBC example
+Formulae presume the following content is saved to the file `fbc.yaml`
+
+```yaml
+---
+defaultChannel: stable-v1.0
+name: testoperator
+schema: olm.package
+---
+entries:
+  - name: testoperator.v1.0.0
+  - name: testoperator.v1.0.1
+    skips:
+      - testoperator.v1.0.0
+name: candidate-v1.0
+package: testoperator
+schema: olm.channel
+---
+entries:
+  - name: testoperator.v1.1.0
+    replaces: testoperator.v1.0.1
+    skips:
+      - testoperator.v1.0.0
+name: candidate-v1.1
+package: testoperator
+schema: olm.channel
+---
+entries:
+  - name: testoperator.v1.0.1
+name: fast-v1.0
+package: testoperator
+schema: olm.channel
+---
+entries:
+  - name: testoperator.v1.1.0
+    replaces: testoperator.v1.0.1
+name: fast-v1.1
+package: testoperator
+schema: olm.channel
+---
+entries:
+  - name: testoperator.v1.0.1
+name: stable-v1.0
+package: testoperator
+schema: olm.channel
+```
+
+### rendered `olm.bundle` FBC example
+Formulae presume the following content is saved to the file `bundle.yaml`
+
+```yaml
+---
+image: quay.io/organization/testoperator:v1.1.0
+name: testoperator.v1.1.0
+package: testoperator
+properties:
+- type: olm.package
+  value:
+    packageName: testoperator
+    version: 1.1.0
+- type: olm.gvk
+  value:
+    group: api.testoperator.io
+    kind: Instance
+    version: v1alpha1
+relatedImages:
+- image: busybox
+  name: ""
+- image: quay.io/organization/testoperator:v1.1.0
+  name: ""
+schema: olm.bundle
+```
+
+## Formulae
+
+### Adding a new bundle to an existing channel
+#### semver
+Add a new `testoperatorv1.1.1` bundle pullspec to the `Candidate` channel archetype
+
+```bash
+yq eval '.candidate.bundles += [{"image" : "quay.io/organization/testoperator:v1.1.1"}]' semver.yaml
+```
+
+produces updated `Candidate` archetype contents:
+
+```yaml
+...
+candidate:
+  bundles:
+  - image: quay.io/organization/testoperator:v1.0.0
+  - image: quay.io/organization/testoperator:v1.0.1
+  - image: quay.io/organization/testoperator:v1.1.0
+  - image: quay.io/organization/testoperator:v1.1.1
```

Comment:
Maybe overkill, but it might be nice to take advantage of the code callouts added in #286. The image values only vary a tiny bit. The code callout may help draw attention to the added line faster for a reader

Reply:
IMHO, because of the fact that the outputs are already focused on the changed elements and the operations are pretty basic, I think it's overkill (though I'm a fan of the callouts in general).

Reply:
Fair enough. Only mentioned it because there are a lot of `1`s in all the version tags so not a huge differentiator to catch the eye but not enough to make me cause a fuss :)